src/Exporter.php

```diff
@@ -58,7 +58,7 @@ public function getResponse(string $format, string $filename, SourceIteratorInte
         $writer = $this->writers[$format];
 
         $callback = function () use ($source, $writer): void {
-            $handler = \Exporter\Handler::create($source, $writer);
```

Comment:
Care with this. Handler namespace is different From Exporter namespace

Reply:
No it isn't they are both at the root of `src`

Reply:
Ohhh it was different on 1.x not on master anymore

Reply:
? they are both in the same namespace on 1.x too? 🤔

Reply:
Yes.. indeed. But I was looking at your Exporter class namespace from master vs Handler namespace from 1.x and it were different. :)

Reply:
Oooh ok alright then :)